portfolio/dash/app.py

```
@@ -623,88 +77,18 @@ def run_collect(n_clicks: int):
+        return f"Refresh error: {exc}"
 
 
 def main() -> None:
-    from portfolio.dash.rcx_callbacks import register_rcx_callbacks
+    from portfolio.dash.overview_tab import register_overview_callbacks
+    from portfolio.dash.rcx_callbacks import register_edge_callbacks
 
-    register_rcx_callbacks(app)
+    register_overview_callbacks(app, THEME)
+    register_edge_callbacks(app)
     app.run(host="0.0.0.0", port=8050, debug=False)
```

Ruff (0.15.15)

[error] Possible binding to all interfaces

(S104)